cmd/e2e/basic_test.go

```diff
@@ -72,7 +72,7 @@ func verifyEDS(t *testing.T, edsName string, edsSpec zv1.ElasticsearchDataSetSpe
 	eds, err := waitForEDS(t, edsName)
 	require.NoError(t, err)
 	err = waitForEDSCondition(t, eds.Name, func(eds *zv1.ElasticsearchDataSet) error {
-		if !assert.ObjectsAreEqualValues(replicas, eds.Spec.Replicas) {
+		if !assert.EqualValues(t, replicas, eds.Spec.Replicas) {
```

Comment:
I'm pretty sure they are timing out and #27 would fix it.

Reply:
But this change also looks fine actually.

Reply:
I'm working on this. Should be easy.